### DISCUSSION TITLE
Is it possible to select avatar in a scene?

### DISCUSSION THREAD

**Discussion by HYPE-STUDIOS-PHILIP** [mozilla/hubs#3038] [General Questions & Discussion] [answered]
Hi!

Is it possible to select an avatar after you've entered a scene. I know you can add avatars when you are in the scene, but is there a way to add avatars that you can select in the editor so the avatars are pre-chosen for the players to select from.

If that made sence?

Thanks

**Comment by ocularvr** [comment, marked as answer]
Absolutely. Have a look at this article:
https://hubs.mozilla.com/docs/sharing-avatar-links-privately.html
Its really easy, just add a link in spoke, make the link url the one you copied, ie the avatar.
You will get a nice picture of the avatar so it does what you want, also think about it, doing it this way saves you lots of bandwidth and processing power. Like this, mobiles, desktops, and even headsets can see. The idea of using the actual avatar mesh might limit this prospect.
Hope this helps :)

**Reply by HYPE-STUDIOS-PHILIP** [reply]
Thank you so much, that work perfectly, just as we needed! :)
